### PR TITLE
Fixed REP comment importer to import full message not just first word

### LINKED ISSUE #178: REP Comment Parser only catching first word
In the REP format for comments, the content for the comment itself is all the tokens that appear after the track name (or narrative-entry type for `;NARRATIVE2:` entries.

But, our current token-based processing only store the first comment. That's a mistake - it should group them together to store a phrase.

_Hopefully_ the REP comment parser uses the ExtractionHighlighter, and this issue gives us a further opportunity to test-drive composite highlight tokens.

### PR DESCRIPTION
**Pull request by robintw**
Fixes #178 

Highlighted output now looks like the attached. Can you check this is right @IanMayo - as you're the expert on REP files. It looks right to me though.
Screen Shot 2020-03-10 at 10 02 03

### CODE REVIEW

File: pepys_import/file/replay_comment_importer.py [hunk 1]
```diff
@@ -108,8 +108,10 @@ def load_this_file(self, data_store, path, file_object, datafile):
                     self.name, "timestamp", timestamp, "n/a"
                 )
 
-                message = message_token.text
-                message_token.record(self.name, "message", message, "n/a")
+                message = " ".join([t.text for t in message_tokens])
+                combine_tokens(*message_tokens).record(
```
Comment (IanMayo) [inline pepys_import/file/replay_comment_importer.py]: Aah, this was the Python that threw me, the use of the `*` operator.

Comment (robintw) [reply pepys_import/file/replay_comment_importer.py]: We could add a comment to explain it if you want? It basically turns a list into a set of arguments to a function. So here if there are five tokens in `message_tokens` then it passes those as five arguments to `combine_tokens`.

**Comment by IanMayo** [comment]
I'll modify the test, to verify the full string has gone into the new record.

File: pepys_import/file/replay_comment_importer.py (hunk 1 above)
Comment (IanMayo) [reply pepys_import/file/replay_comment_importer.py]: No, it's ok @robintw .  We have to assume a basic level of Python skills.

**Pull request merged**